src/styles/components/dropdown.scss

```diff
@@ -82,8 +82,10 @@ $disabled-dropdown-item: var(--disabled-dropdown-item);
 			filter: brightness(90%);
 		}
 	}
-	& .dropdown-item span {
-		color: $text-dropdown-color;
+	& .dropdown-item {
```

Comment:
@Aringoaway This will affect also other dropdowns, are you sure you want to do this? There is no description of this PR, which will say how the component will be affected...

Reply:
@Pe5h4 this styles for TenantDetailContainer and RolesDetailContainer when i implemented Credential component, this component has <a> and <i> tag, and <a> tag had a blue color, <i> had a white color in dark mode. Because of it i added color styles in dropdown.scss

Reply:
@Aringoaway But what if you want to preserve `Link` in the dropdown in some other cases than for TenantDetail and RolesDetail? Would not it be better to style it in the scss of TenantDetail and RolesDetail containers directly rather than creating a global style?

Reply:
@Pe5h4 I thought it would be better to do it globally, because in case there are similar cases later. But I don't know how to do it better, I can write those styles locally

Reply:
Sorry that I might have stupid questions, but its difficult for me to orient myself between PR's without proper description/explanation of what has been changed and why and how it will e.g. affect other components

Reply:
@Aringoaway Sorry, I am bit tired and overwhelmed, I missed your explanation above https://github.com/TeskaLabs/asab-webui/pull/342#discussion_r937784741

In that case it makes sense